modules/swagger-parser/src/test/java/io/swagger/parser/SwaggerParserTest.java

```diff
@@ -468,15 +469,7 @@ public void testIssue292WithCSVCollectionFormat() {
         QueryParameter qp = (QueryParameter) param;
         assertEquals(qp.getCollectionFormat(), "csv");
     }
-
-    @Test
-    public void testIssue255() {
-        SwaggerParser parser = new SwaggerParser();
-
```

Comment:
Hi, why delete this test instead of making it work with the new approach?.

Reply:
Could I try to make this test work? Again, anything to move this fix along would be great.

Reply:
yes, please and rebase as well. Thanks!

Reply:
So, grab the pull request, fix the test, rebase against the latest, and resubmit the pull request?  You know I'm not the original author of this pull request ...

Reply:
oh ok. So how else can we speed this fix?

Reply:
I'll try to patch up the pull request and see if I can make the test work.  Hopefully sometime later today or tomorrow (unless the original author fixes this first).

Reply:
After some work/investigation and consulting the original issue 255, I believe this fix is correct and the original was not (and this is also indicated/hinted at in the fix of 255).  There are unit tests in this fix that verify that a proper ArrayModel is returned for the array example (not a string as in the prior fixes unit test).  I also tested with several downstream pipelines (one that takes swagger -> markdown -> asciidoc -> pdf and a similar one that takes swagger -> markdown -> asciidoc -> static html).  In each case, the fixed version works (i.e. the final pdf / html with an array of objects is displayed correctly).  So, the test was removed because it is somewhat redundant.  Also, it seems like you might want the original issue 255 authors/collaborators to weigh in as I do not know what their downstream usage of the swagger-parser is (so this fix could break their usage - I cannot tell).

Reply:
So, are we good to go?  Will this show up soon in a release?

Reply:
Hi we need this updated with master branch, I do not have permission to update the @boillodmanuel PR.

Reply:
@kidnme merged!